### DISCUSSION TITLE
Support for higher dbtcore version

### DISCUSSION THREAD

**Discussion by lrcuwicRSG1** [dbt-msft/dbt-sqlserver#459] [General]
Hi @all.
I have a question: Currently dbt-sqlserver project supports dbtcore-1.4.9 --> Is there  a plan to upgrade to current dbt-core versions (e.g. 1.7.3) --> dbtcore-1.4.9 will run out of critical support on 25-01-2024. See https://endoflife.date/dbt-core
Thanks

**Comment by openfarma** [comment]
I think this is abandonware.

**Comment by cody-scott** [comment]
@lrcuwicRSG1 not sure if you're still interested in this or not. Not abandoned; working on updating to 1.8 (currently ~1.7)
